tests/Doctrine/Tests/DBAL/Sharding/SQLAzure/AbstractTestCase.php

```diff
@@ -44,6 +44,7 @@ protected function setUp()
         // Customers, Orders, OrderItems federation tables.
         // See http://cloud.dzone.com/articles/using-sql-azure-federations
         $this->sm = new SQLAzureShardManager($this->conn);
+        var_dump($tmpConn->getSchemaManager()->listDatabases());
```

Comment:
Nope there is no `$tmpConn` here. I wanted you to add it to `TestUtils` class, please have a second look at what I wrote ;)